otterdog/eclipse-edc.jsonnet

```diff
@@ -458,6 +458,29 @@ orgs.newOrg('eclipse-edc') {
         },
       ],
     },
+    orgs.newRepo('Technology-HuaweiCloud') {
+      allow_rebase_merge: false,
+      allow_update_branch: false,
+      delete_branch_on_merge: false,
+      has_discussions: true,
+      has_wiki: false,
+      squash_merge_commit_title: "PR_TITLE",
+      web_commit_signoff_required: false,
+      workflows+: {
+        default_workflow_permissions: "write",
```

Comment:
it looks like you copied an existing repo, which is perfectly fine, though for new repos we would like to already use default workflow permissions set to read as this should become the default for all repos ideally at some point in time. While workflows in existing repos might be some effort, creating new workflows in new repos should already default to the new fine grained permission model that GitHub supports imho.